olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml

```
@@ -214,7 +214,7 @@ spec:
             displayName: Version
             path: version
         version: v1beta1
-      - description: Represents an installation of a particular version of Knative Kafka components
+      - description: Apache Kafka focused extensions for Knative Eventing
```

```suggestion
      - description: Apache Kafka based extensions for Knative Eventing
```
I feel like "Apache Kafka _focused_ extensions" implies that the extensions are focused on targetting Kafka, while "Apache Kafka _based_ extensions" implies that the extensions are based on Kafka.